tailwind.config.js

```diff
@@ -2,6 +2,7 @@ const colors = require('tailwindcss/colors')
 
 export default {
   content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
+  darkMode: 'class',
```

Comment:
Nice feature of Tailwind. This makes it so that the `dark:` utility class is looking for anytype of `dark` class applied to a parent element. In this case, I'm just simply toggling the `html` node to have `class="dark"` and that is what handles the actual theme transition.

If this were to be omitted, Tailwind by default would use the user's `prefers-color-scheme: ` value to determine the theme, without any way of toggling the theme.